app/views/unattended/report_templates/host_statuses_csv.erb

```diff
@@ -3,7 +3,12 @@ name: Host statuses CSV
 snippet: false
 model: ReportTemplate
 -%>
-Name,Global,<%= all_host_statuses.map { |s| s.status_name }.join(',') %>
 <%- load_hosts(includes: :host_statuses).each_record do |host| -%>
-<%=   host.name -%>,<%= host.global_status -%>,<%= all_host_statuses.map { |s| host_status(host, s.status_name).status }.join(',') %>
+<%- report_row(
```

Comment:
this is not indented correctly, it should have extra two spaces since it's inside the load_hosts block
`<%- report_row(`
vs
`<%-   report_row(`

also I think the closing erb tag could be moved, how about this?

```
<%#
name: Host statuses CSV
snippet: false
model: ReportTemplate
-%>
<%- load_hosts(includes: :host_statuses).each_record do |host| -%>
<%-   report_row({
        'Name': host.name,
        'Global': host.global_status
      }.merge(all_host_statuses_hash(host))) -%>
<%- end -%>
<%= report_render %>
```

I'd say it slightly more readable